homework/java/alexandr_raby/MathOperation/aTemplate/src/main/java/by/jrr/start/MahOperationDemo.java

```diff
@@ -0,0 +1,64 @@
+package by.jrr.start;
+
+public class MahOperationDemo {
+
+    public static void main(String[] args){
+        System.out.println("Welcome to the 'Math Operations'!");
+
+        int a = 90;
+        int b = 40;
+
+        System.out.println("1) Addition '+':");
+        System.out.print("a = ");
+        System.out.print(a);
+        System.out.print(", ");
+        System.out.print("b = ");
+        System.out.print(b);
```

Comment:
if you write "println" there you don't need to write "System.out.println()" later.